frame/db_schema.ts

```diff
@@ -9,13 +9,14 @@
  * IndexedDB, with runtime type checking.
  */
 
+import { Ad, InterestGroup } from "../lib/shared/api_types";
 import { isArray } from "../lib/shared/guards";
 import { useStore } from "./indexeddb";
 
 /** An `Ad` from the public API serialized into storage format. */
-export type Ad = [renderingUrl: string, price: number];
+type AdRecord = [renderingUrl: string, price: number];
```

Comment:
You don't need to change it in this PR, but it's weird having the price pulled out into the storage format, since that isn't actually part of the concept of an interest group (just our handling of not having bedding functions yet).  I think it would make more sense to directly store the user provided metadata.

Reply:
The static price is going away in a future PR, now that we've decided we're not going to compile any bidding or scoring functions into the frame.